main.go

```diff
@@ -33,14 +33,21 @@ func whoAmI() string {
 }
 
 func main() {
+	var BuildVersion string
```

Comment:
I think this does need to be a global variable rather than a variable that's local to the main function.

After you've moved the variable out, can you test it by building locally (with the appropriate ldflags) and running `alpaca -v`?

Reply:
Will do

Reply:
https://github.com/samuong/alpaca/pull/81/commits/ba054bfb488e9c0010270446e0bdd9bf6c73c640